rc/lsp.kak

```diff
@@ -2234,7 +2261,7 @@ foreach my $i (0 .. $#sel_content) {
         $def =~ s/'\''/'\'''\''/g;
         print(" '\''$def'\''");
     } else {
-        print(" '\'''\''");
+        print(" '\'' '\''");
```

Comment:
Ok this means we now always have a non-empty placeholder text? Sounds great, this can potentially simplify `lsp-snippets-select-next-placeholders`

Reply:
yes - and i hope to do that as well at some point